utils/utils_test.go

```diff
@@ -0,0 +1,58 @@
+package utils
+
+import (
+	"github.com/stretchr/testify/assert"
+	"os"
+	"testing"
+)
+
+const (
+	defaultHostCNIBinPath = "/host/opt/cni/bin"
+	defaultVethPrefix     = "eni"
+	defaultMTU            = "9001"
+	defaultEnablePodEni   = "false"
+
+	envHostCniBinPath = "HOST_CNI_BIN_PATH"
+	envVethPrefix     = "AWS_VPC_K8S_CNI_VETHPREFIX"
+	envEniMTU         = "AWS_VPC_ENI_MTU"
+	envEnablePodEni   = "ENABLE_POD_ENI"
```

Comment:
You should just use a fake environment variable and default value for it here. One should be enough for testing, and a fake value keeps it simple